faq/billing.mdx

```diff
@@ -52,6 +52,16 @@ Below is a detailed list of what is considered as start and stop billing events
 
 <Concept>
 
+## What does pro-rata billing for Dedibox and Web Hosting services mean?
+
+In the initial month of a subscription, you are billed pro-rata in the following month. This means the charge is adjusted to reflect the portion of the first month you used the service.
```

Comment:
```suggestion
You care billed pro-rata for the initial month of a subscription. This means the charge is adjusted to reflect the portion of the first month you used the service.
```

Reply:
I think it's confusing to point out the bill arrives in the following month, maybe we can just eliminate the mention of when the pro rata bill for the first month actually arrives?